service/installer/installer.go

```diff
@@ -88,9 +88,12 @@ func New(config Config) (spec.Installer, error) {
 		helmConfig.FileSystem = config.FileSystem
 		helmConfig.Logger = config.Logger
 
+		helmConfig.Environment = config.Viper.GetString(config.Flag.Service.Deployer.Environment)
 		helmConfig.HelmBinaryPath = config.Viper.GetString(config.Flag.Service.Deployer.Installer.Helm.HelmBinaryPath)
 		helmConfig.Organisation = config.Viper.GetString(config.Flag.Service.Deployer.Installer.Helm.Organisation)
 		helmConfig.Password = config.Viper.GetString(config.Flag.Service.Deployer.Installer.Helm.Password)
+		helmConfig.PollInterval = config.Viper.GetDuration(config.Flag.Service.Deployer.Eventer.GitHub.PollInterval)
```

Comment:
To make it simpler, I would like to make poll interval as same as GitHub event.

Reply:
If we want things simple: do we even need to externally configure that? Can't you keep it as a const inside the package?

Reply:
Right, I guess we are not going to change to poll interval very often, or varied each installation. So const suggestion looks fine.

Reply:
Done to `const`, @kopiczko PTAL